doc/text/news.md

```diff
@@ -1,5 +1,12 @@
 # News
 
+## 1.0.10 - 2019-12-08
```

Comment:
Generally, you don't need to update this file.

We don't release new version at 2019-12-08.

Reply:
ca617f7